docs/src/references/supported-platforms.md

```diff
@@ -6,24 +6,92 @@ sidebar_position: 15
-* minimum 16MB of RAM
-* mosquitto minimum version 1.6 (for security reasons we recommend the latest 1.x version)
+thin-edge.io can be run on any Linux based Operating System, such as; Debian/Ubuntu, Alpine, RHEL, Fedora, Pokey (Yocto) etc.
 
-:::info
-Systemd is supported out-of-the box and is recommended for production systems. Other init systems (e.g. OpenRC, s6-overlay, SysVInit, supervisord, runit) are supported via addition packages as detailed in the [init system](../operate/installation/init_systems.md) section.
+In addition to a Linux based Operation system, the following requirements must also be fulfilled:
+
+* At least ~40MB of RAM (see [notes](#memory-usage))
+* [mosquitto](https://github.com/eclipse/mosquitto) MQTT Broker (see [notes](#recommended-mosquitto-version))
+
+:::tip
+If you are looking for new hardware it is highly recommended to choose a CPU which has at least 2 cores, as it provides a much more responsive experience and should provide enough processing headroom for future requirements.
+:::
+
+:::note
+mosquitto will be installed via the installation script, however it needs to be available from a software repository, however mosquitto is generally available in most popular Linux based operating systems.
 :::
 
-# Level 1
-Level 1 supported platforms are officially supported and are actively tested in the CI/CD.
-* ARMv7 Raspberry Pi OS 10
-* ARMv8 Raspberry Pi OS 10
-* AMD64 Ubuntu 20.04
-
-# Level 2
-Level 2 platforms are not officially supported and tested yet, but we know from our experiences that these systems used to work for some maintainers or users. If your os is not listed here, this does not mean it is not working, just give it a try. We are happy to hear about your experience in the Github discussions.
-* Ubuntu 20.04 in WSL (only for development, not for running thin-edge.io due to missing systemd)
-* AMD64 Debian 10
-* ARMv6 Raspberry Pi OS 10 (needs to be built for this specific target, please refer to [Issue-161](https://github.com/thin-edge/thin-edge.io/issues/161))
-* ARMv7 Raspberry Pi OS 11
-* ARMv8 Raspberry Pi OS 11
+## Recommended mosquitto version
+
+Whilst thin-edge.io will work with any mosquitto version, it is recommended to use mosquitto version &gt;=2.0.18 as it includes some important fixes regarding the MQTT bridge functionality.
+
+If you can't use the recommended version, then be aware of the following issues which affect various mosquitto versions in some corner cases:
+
+|Mosquitto Version|Issues|
+|-----------------|------|
+|2.0.11|https://github.com/eclipse/mosquitto/issues/2604|
+|2.0.15|https://github.com/eclipse/mosquitto/issues/2634|
+
+## Supported CPU Architectures
+
+thin-edge.io is developed in Rust which is an ahead-of-time compiled language. This means that not only does thin-edge.io need to be compiled to produce the executable, that each executable is specific for different CPU Architectures (e.g. x86_64/amd64, aarch64/arm64, armv7, armv6 etc.).
```

Comment:
Isn't this common knowledge?

```suggestion
thin-edge.io is developed in Rust which is a compiled language and each executable is specific for different CPU Architectures (e.g. x86_64/amd64, aarch64/arm64, armv7, armv6 etc.).
```

Reply:
I wouldn't assume this is common knowledge...especially if people are coming from the python world where generally the user does not have to think about the CPU architecture...but I'll reword as per your suggestion.